Current set up install cloning fails.

Same here `npm install` fails

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: next@13.4.9-canary.1
npm ERR! node_modules/next
npm ERR!   next@"13.4.9-canary.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer next@"^12.2.5 || ^13" from next-auth@4.22.1
npm ERR! node_modules/next-auth
npm ERR!   next-auth@"4.22.1" from the root project
npm ERR!   peer next-auth@"^4" from @next-auth/prisma-adapter@1.0.7
npm ERR!   node_modules/@next-auth/prisma-adapter
npm ERR!     @next-auth/prisma-adapter@"^1.0.5" from the root project
```

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "js-cookie": "^3.0.1",
     "lucide-react": "^0.244.0",
     "nanoid": "^4.0.2",
-    "next": "13.4.9-canary.1",
+    "next": "13.4.9",
     "next-auth": "4.22.1",
     "next-mdx-remote": "^4.4.1",
     "openai-edge": "^1.1.0",

